jobs/ci-kubernetes-e2e-cos-docker-validation-serial.env

```diff
@@ -1,5 +1,5 @@
-# Use GCI builtin k8s version.
-JENKINS_USE_GCI_VERSION=y
+# Use fixed k8s version.
+JENKINS_PUBLISHED_VERSION=v1.6.1
```

Comment:
Thanks fejta@, but here we want to test what's builtin into COS, which is 1.6.1 currently, 2 version away from `ci/latest-1.6`.